src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/FileIO/FileSystem.vb

```diff
@@ -2223,7 +2223,7 @@ Namespace Microsoft.VisualBasic.FileIO
 
                 ' If user wants to ignore case, convert new string to lower case. m_SearchText was converted in constructor.
                 If m_IgnoreCase Then
-                    Return New String(CharBuffer).ToUpper(CultureInfo.CurrentCulture).Contains(m_SearchText)
+                    Return New String(CharBuffer).Contains(m_SearchText, StringComparison.CurrentCultureIgnoreCase)
```

Comment:
Not sure if this is safe CC @tarekgh

Reply:
There will be a potential change, but the original code was doing casing using culture then comparing the strings as ordinal which is strange. Considering that it would be better to go with `Return New String(CharBuffer).Contains(m_SearchText, StringComparison.OrdinalIgnoreCase)`.

Reply:
Thanks, updated